CHANGELOG.md

```diff
@@ -13,6 +13,9 @@ For older changes see the [archived Singularity change log](https://github.com/a
   working directory, though `--pwd` is still supported for compatibility.
 - When building RPM, we will now use `/var/lib/apptainer` (rather than
   `/var/apptainer`) to store local state files.
+- The `--vm` and related flags to start apptainer inside a VM have been
+  removed. This functionality was related to the retired Apptainer Desktop /
```

Comment:
Please change this Apptainer to Singularity.  Those projects were retired before the name change.